js/ui/html_editor/utils/image_uploader_helper.js

```diff
@@ -533,3 +528,11 @@ export function urlUpload(quill, index, attributes) {
     quill.insertEmbed(index, 'extendedImage', attributes, USER_ACTION);
     quill.setSelection(index + 1, 0, USER_ACTION);
 }
+
+export function serverUpload(url, fileName, quill, pasteIndex) {
+    if(url) {
```

Comment:
Earlier we had a condition for `!this.useBase64` here. Does this case work ok now?

Reply:
When we check checkbox then it uses base64 'branch' in the code and serverUpload method is not called